swan-cern/scripts/before-notebook.d/10_spark.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Author: Danilo Piparo, Enric Tejedor, Pedro Maximino, Diogo Castro 2023
+# Copyright CERN
+# The environment for Spark is configured here.
+
+if [[ ! -n $SPARK_CLUSTER_NAME ]]
+then
+  # Disable spark jupyterlab extensions enabled by default if no cluster is selected
+  mkdir -p /etc/jupyter/labconfig
+  echo "{
+    \"disabledExtensions\": {
+      \"sparkconnector\": true,
+      \"@swan-cern/hdfsbrowser\": true,
```

Comment:
We don't install hdfsbrowser anymore, so this should be removed.

Reply:
This was removed.